src/absil/il.fs

```diff
@@ -8,14 +8,18 @@ module (*internal*) Microsoft.FSharp.Compiler.AbstractIL.IL
 
 
 open Internal.Utilities
+#if FABLE_COMPILER
```

Comment:
You may as well make this unconditional e.g.

    open Microsoft.FSharp.Collections // needed for Fable
    open Microsoft.FSharp.Core // needed for Fable